packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 1.28.0 - (July 31, 2023)
+
+* Changed
+  * Minor dependency version bump
+
 ## 1.26.0 - (July 20, 2023)
```

Comment:
Can you remove this line? Sorry this got added from a merge conflict by https://github.com/cerner/terra-framework/pull/1710.

Reply:
[3b02a86](https://github.com/cerner/terra-framework/pull/1720/commits/3b02a863852655bc86a0cdd9a52a0801f17b27b5) Removed